docs/data-sources/overlay.md

```diff
@@ -642,6 +644,8 @@ data "kustomization_overlay" "minecraft" {
     release_name = "moria"
     include_crds = false
     skip_tests = false
+		kube_version = "1.29.0"
+		api_versions = ["monitoring.coreos.com/v1"]
```

Comment:
Shouldn't this be aligned with the others?

Reply:
Fixed the indention, there was a tab instead on spaces.